appveyor.yml

```diff
@@ -60,23 +60,19 @@ install:
 
   # Install the dependencies of the project.
   - ps: Add-AppveyorMessage "Installing conda packages..."
-  - "%CMD_IN_ENV% conda install -yq %TEST_DEPS%"
-  - "%CMD_IN_ENV% conda install -yq %DEPS%"
-  - "pip install pytest-mpl"
-
-  # Having 'sip' folder on path confuses import of `sip`.
-  #- "%CMD_IN_ENV% conda install -yq pip"
+  - "conda install -yq -c conda-forge %TEST_DEPS%"
+  - "conda install -yq -c conda-forge %DEPS%"
 
   - ps: Add-AppveyorMessage "Installing hyperspy..."
-  - "python setup.py install"
+  - "pip install .[all]"
 
 build: false  # Not a C# project, build stuff at the test step instead.
 
 test_script:
   # Run the project tests
   - ps: Add-AppveyorMessage "Running tests..."
   - "python setup.py build_ext --inplace"
-  - "py.test --mpl"
+  - "pytest --mpl"
```

Comment:
May this be the reason why pytest fails in appveyor for 3.5?

Reply:
Non, I have tried... I will revert all the changes that I made on CI, because at the moment conda doesn't resolve nicely the dependencies with python 3.5. I guess the main reason is that a few packages aren't build anymore for python 3.5 in conda-forge.
When we add support for python 3.7, we should revisit tidying up CI and possibly also adding testing with numpy and matplotlib development.